docs/src/userguide/freethreading.rst

```diff
@@ -160,3 +160,297 @@ to require extreme care to make it work correctly).
    (i.e. all in one go, without switching to another thread) on non-free-threaded builds.
    Many operations can cause the GIL to be released. Some more detail is in the section
    :ref:`gil_as_lock`.
+
+
+Opinionated Suggestions
+=======================
+
+This section contains our views on how to use Cython effectively with free-threaded
+Python.  It may evolve as our understanding grows.
+
+Interaction between threads
+---------------------------
+
+Multi-threaded programs general work best if you can minimize the interaction between
+threads for example by having the different threads perform completely isolated
+blocks of work which are only collected at the end.  Python code is no exception -
+especially since Python reference counting means that even apparently "read-only"
+operations usually actually involve both reading and writing.
```

Comment:
```suggestion
Multi-threaded programs generally work best if you can minimize the interaction between
threads. It's optimal if the different threads perform completely isolated
blocks of work which are only collected at the end.  Python code is no exception -
especially since Python's reference counting means that even apparent "read-only"
operations can actually involve both reading and writing.
```